core/math/math_funcs.h

```diff
@@ -105,6 +105,9 @@ class Math {
 	static _ALWAYS_INLINE_ double fmod(double p_x, double p_y) { return ::fmod(p_x, p_y); }
 	static _ALWAYS_INLINE_ float fmod(float p_x, float p_y) { return ::fmodf(p_x, p_y); }
 
+	static _ALWAYS_INLINE_ double modf(double p_x, double &p_int) { return ::modf(p_x, &p_int); }
+	static _ALWAYS_INLINE_ float modf(float p_x, float &p_int) { return ::modff(p_x, &p_int); }
+
```

Comment:
```suggestion
	static _ALWAYS_INLINE_ double modf(double p_x, double *p_int) { return ::modf(p_x, p_int); }
	static _ALWAYS_INLINE_ float modf(float p_x, float *p_int) { return ::modff(p_x, p_int); }

```
Wouldn't this be more natural given cpp's modf format?

Reply:
I wrote it in a more C++ form of way since the std library is C compatible. But I am fine with the proposed changes if you insist? Having it as a pointer indicates better that it is an "out" value.